### PR TITLE
test: odp_pktio_ordered: add missing schedule_config initialization

### PR DESCRIPTION
**Pull request by matiaselo**
Signed-off-by: Matias Elo <matias.elo@nokia.com>
Reported-by: Jari Mustajärvi <jari.mustajarvi@nokia-bell-labs.com>

### CODE REVIEW

File: test/performance/odp_pktio_ordered.c [hunk 1]
```diff
@@ -1130,7 +1130,8 @@ int main(int argc, char *argv[])
 	/* Parse and store the application arguments */
 	parse_args(argc, argv, &gbl_args->appl);
 
-	odp_schedule_config(NULL);
+	odp_schedule_config_init(&schedule_config);
+	odp_schedule_config(&schedule_config);
```
Comment (Bill-Fischofer-Linaro) [inline test/performance/odp_pktio_ordered.c]: Why this change? Aren't these equivalent sequences?

**Comment by muvarov** [comment]
Merged.
